lib/ui/common/roundedButton.dart

```diff
@@ -88,15 +73,15 @@ class RoundedButton {
     }
 
     return GestureDetector(
-      onTap: () => _showToast(viewModel.context),
+      onTap: () => viewModel.onTap(),
       child: Padding(
         padding: buttonStyle.edgePadding,
         child: Container(
           alignment: Alignment.center,
           height: buttonStyle.size,
           width: buttonStyle.size,
           decoration: BoxDecoration(
-              color: buttonStyle.backgroundColor,
+              color: viewModel.backgroundColor != null ? viewModel.backgroundColor : buttonStyle.backgroundColor,
```

Comment:
why viewModel?

Reply:
RoundedButton modifications has been done in Farm-62 view prof loss statement's PR.